src/IssueViz.Security/OpenInIDE/OpenInIDEResources.resx

```diff
@@ -132,6 +132,26 @@
   <data name="Factory_Succeeded" xml:space="preserve">
     <value>[Open in IDE] Request listener created successfully. Listening on port {0}.</value>
   </data>
+  <data name="Infobar_InvalidStateReason_NotInConnectedMode" xml:space="preserve">
```

Comment:
I'm constructing the final message from multiple resources, so they would be:
`Could not handle Open in IDE request: <reason>. <insturctions>`
E.g.
`Could not handle Open in IDE request: you must connect to the right server. Currently connected to: https://sq-server.com. Please connect to https://sonarcloud.io/, organization sonarsource, and bind to project 'someproject'.`

Please review the messages and I'll share them with the rest of the team.

Reply:
That looks like a lot of text to fit in the goldbar. How does it look in the IDE?

Reply:
If the solution is currently in connected mode but not bound to the requested server/org/project, then isn't it more likely that the action the user will need to take is to open a different solution, rather than change the current binding? At the moment, the wording seems to imply changing the binding.

Reply:
Regarding the wording: hmm I can see that now. What would you suggest? 
Maybe instead of `Please connect to..` - `Please open a solution that is connected to..`?